src/strategies_and_params/reportingstrategy.jl

```diff
@@ -154,28 +165,31 @@ function print_stats(io::IO, step, state)
 end
 
 """
-    ReportDFAndInfo(; k=1, i=100, io=stdout, writeinfo=false) <: ReportingStrategy
+    ReportDFAndInfo(; reporting_interval=1, info_interval=100, io=stdout, writeinfo=false) <: ReportingStrategy
 
-The default [`ReportingStrategy`](@ref). Report every `k`th step to a `DataFrame` and write
-info message to `io` every `i`th step (unless `writeinfo == false`). The flag `writeinfo` is
-useful for controlling info messages in MPI codes, e.g. by setting
+The default [`ReportingStrategy`](@ref). Report every `reporting_interval`th step to a `DataFrame` 
+and write info message to `io` every `info_interval`th reported step (unless `writeinfo == false`). The flag 
+`writeinfo` is useful for controlling info messages in MPI codes, e.g. by setting
 `writeinfo =`[`is_mpi_root()`](@ref).
 """
 @with_kw struct ReportDFAndInfo <: ReportingStrategy
-    k::Int = 1
-    i::Int = 100
+    reporting_interval::Int = 1
+    info_interval::Int = 100
     io::IO = stdout
     writeinfo::Bool = false
 end
 function report!(s::ReportDFAndInfo, step, args...)
```

Comment:
```suggestion
report!(::ReportDFAndInfo, _, args...) = report!(args...)
```
Checking `s.reporting_interval` isn't necessary at this point, because the `if` condition in `lomc!` and `advance!` control whether reporting is necessary.